wled00/overlay.cpp

```diff
@@ -89,6 +89,15 @@ void _overlayAnalogCountdown()
 
 void handleOverlayDraw() {
   usermods.handleOverlayDraw();
+  if (analogClockSolidBlack) {
+    Segment* segments = strip.getSegments();
+    for (uint8_t i = 0; i < strip.getActiveSegmentsNum(); i++) {
+      Segment segment = segments[i];
```

Comment:
That's unnecessary segment object copy. Use a reference instead.

Reply:
Done